engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -91,6 +91,23 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
       }
     }
 
+  def submitBatchRoute =
+    path("workflows" / Segment / "batch") { version =>
+      post {
+        formFields("wdlSource", "workflowInputs".?, "workflowOptions".?) {
+          (wdlSource, workflowInputs, workflowOptions) =>
+            requestContext =>
+              import spray.json._
+              workflowInputs.getOrElse("[]").parseJson match {
+                case JsArray(inputses) =>
+                  val sources = inputses.map(inputs => WorkflowSourceFiles(wdlSource, inputs.compactPrint, workflowOptions.getOrElse("{}")))
```

Comment:
Thinking out loud: I take it we never need to support variation in workflowOptions for each input set?